src/views/UserDashboard.vue

```diff
@@ -108,36 +141,44 @@ export default {
       return this.appStore.user.username
     },
     displayTodayStats() {
-      return (this.userTodayPriceCount > 0) || (this.userTodayProofCount > 0)
+      if (this.currentTab === 'consumption') {
+        return (this.userTodayConsumptionPriceCount > 0) || (this.userTodayConsumptionProofCount > 0)
+      }
+      // community
+      return (this.userTodayCommunityPriceCount > 0) || (this.userTodayCommunityProofCount > 0)
     },
     displayedPriceList() {
-      if (!this.$vuetify.display.smAndUp) {
-        return this.userPriceList.slice(0, 5)
-      } else {
-        return this.userPriceList
+      if (this.currentTab === 'consumption') {
+        return (!this.$vuetify.display.smAndUp) ? this.userConsumptionPriceList.slice(0, 5) : this.userConsumptionPriceList
+      }
+      // community
+      return (!this.$vuetify.display.smAndUp) ? this.userCommunityPriceList.slice(0, 5) : this.userCommunityPriceList
+    },
+    getPriceParams() {
+      let defaultParams = { owner: this.username }
+      defaultParams['proof__type'] = (this.currentTab === 'consumption') ? 'RECEIPT' : 'PRICE_TAG'
```

Comment:
the `proof__type` filter is not yet available in the backend...

Reply:
unlocked here: https://github.com/openfoodfacts/open-prices/pull/741

Reply:
well actually we also needed this ^^
https://github.com/openfoodfacts/open-prices/pull/766